SampleDanger/ViewController.swift

```diff
@@ -12,8 +12,16 @@ class ViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
+     print("checkdfgh")
+     print("checkdfgh")
+     print("checkdfgh")
+     print("checkdfgh")
+     print("checkdfgh")
+     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 20;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)